commands/commands.go

```diff
@@ -19,6 +23,106 @@ type Cmd struct {
 	Template string
 }
 
+type TomlConfig struct {
+	Env   string
+	Dev   ConsulFromFile
+	Prod  ConsulFromFile
+	Stage ConsulFromFile
+	QA    ConsulFromFile
+	West  ConsulFromFile
+	East  ConsulFromFile
+}
+
```

Comment:
What about a `map[string]ConsulFromFile` instead of hardcoding environment names?